src/lib/notifee.ts

```diff
@@ -41,37 +41,39 @@ export function displayNotification(
 }
 
 export function displayNotificationFromModel(
-  notif: NotificationsFeedItemModel,
+  notification: NotificationsFeedItemModel,
```

Comment:
This can certainly be dropped, but I don't see any particular reason to abbreviate it given that the function is `displayNotificationFromModel`...